### PR TITLE
[Fix] Chart Block language

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

Our current `loadCharts` function in HURUmap/Flourish blocks loads only english charts. Editors can not replace multi lang charts because of this. This PR detects the current post language from permalink and adds the query `lang=` to the `loadCharts` function

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Desktop Screenshots
Peek 2020-03-18 13-17

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: wp-content/plugins/hurumap/src/HURUmapBlock/EditChart.js [hunk 1]
```diff
@@ -48,6 +51,8 @@ function EditChart({
   },
   setAttributes
 }) {
+  const language = useSelect(select => setPostLanguage(select), []);
```
Comment (karimkawambwa) [inline wp-content/plugins/hurumap/src/HURUmapBlock/EditChart.js]: Minor:
```suggestion
  const language = useSelect(select => getPostLanguage(select), []);
```

**Review by karimkawambwa — APPROVED** [review]
LGTM 🚀

**Review by kilemensi — COMMENTED** [review]
:+1: @KhadijaMahanga 

------------------------------

Can't seem to run this locally:
Screenshot from 2020-03-12 18-17-31

But, if `getPostLanguage` is a hook, then it should be [named](https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook) `useXXX` ... otherwise looks good.

**Review by kilemensi — APPROVED** [review]
:rocket: @KhadijaMahanga 

-----------------------------------

Was the issue fixed by just changing to `wordpress/data/select`?

**Comment by KhadijaMahanga** [comment]
>  @KhadijaMahanga
> 
> Was the issue fixed by just changing to `wordpress/data/select`?

Yes, partly!

**Pull request merged**